development/includes/blog-column.pug

```diff
@@ -0,0 +1,73 @@
+<!-- Blog preview link boxes -->
```

Comment:
Please remove these pug include files changes from this pull request since you have them in another one.